src/main/java/uk/gov/companieshouse/api/strikeoffobjections/chips/ChipsClient.java

```diff
@@ -1,23 +1,37 @@
 package uk.gov.companieshouse.api.strikeoffobjections.chips;
 
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.ResponseEntity;
 import org.springframework.stereotype.Component;
+import org.springframework.web.client.RestTemplate;
 import uk.gov.companieshouse.api.strikeoffobjections.common.ApiLogger;
 import uk.gov.companieshouse.api.strikeoffobjections.model.chips.ChipsRequest;
 
 @Component
 public class ChipsClient {
 
-    private ApiLogger apiLogger;
+    private final ApiLogger apiLogger;
+    private final RestTemplate restTemplate;
+
+    @Value("${OBJECT_TO_STRIKE_OFF_CHIPS_REST_INTERFACE_URL}")
+    private String chipsRestUrl;
 
     @Autowired
-    public ChipsClient(ApiLogger apiLogger) {
+    public ChipsClient(ApiLogger apiLogger, RestTemplate restTemplate) {
         this.apiLogger = apiLogger;
+        this.restTemplate = restTemplate;
     }
 
     public void sendToChips(String requestId, ChipsRequest chipsRequest){
-        // TODO OBJ-240 add contact model object and send to CHIPS
         apiLogger.infoContext(requestId,
                 String.format("Posting %s to chips rest interfaces", chipsRequest));
+
+        ResponseEntity<String> chipsRestResponse = restTemplate.postForEntity(chipsRestUrl, chipsRequest, String.class);
+
+        apiLogger.infoContext(
+                requestId,
+                String.format("Sent data to CHIPS, received status code: %s", chipsRestResponse.getStatusCode())
```

Comment:
would it be worth including the url in the log message as well? might help us debug when we have to look through live logs.

Reply:
Added to the log message before the data is sent, as if the request fails this log may not be executed